src/test/java/io/jenkins/plugins/analysis/core/model/DeltaReportTest.java

```diff
@@ -0,0 +1,124 @@
+package io.jenkins.plugins.analysis.core.model;
+
+import java.util.Optional;
+
+import org.apache.commons.lang.StringUtils;
+import org.junit.jupiter.api.Test;
+
+import edu.hm.hafner.analysis.Issue;
+import edu.hm.hafner.analysis.IssueBuilder;
+import edu.hm.hafner.analysis.Report;
+import edu.hm.hafner.analysis.Severity;
+
+import hudson.model.Run;
+
+import io.jenkins.plugins.analysis.core.util.IssuesStatistics;
+import io.jenkins.plugins.analysis.core.util.IssuesStatisticsAssert;
+import io.jenkins.plugins.analysis.core.util.IssuesStatisticsBuilder;
+
+import static io.jenkins.plugins.analysis.core.model.DeltaReportAssert.*;
+import static org.mockito.Mockito.*;
+
+/**
+ * Tests the class {@link DeltaReport}.
+ *
+ * @author Andreas Pabst
+ */
+class DeltaReportTest {
+    private static final String REFERENCE_BUILD_ID = "refBuildId";
+
+    @Test
+    void shouldHaveEmptyReports() {
+        History history = mock(History.class);
+        when(history.getBuild()).thenReturn(Optional.empty());
+        Report report = new Report();
+
+        DeltaReport deltaReport = new DeltaReport(report, history, 0);
+        assertThat(deltaReport)
+                .isEmpty()
+                .hasNoAllIssues()
+                .hasNoOutstandingIssues()
+                .hasNoNewIssues()
+                .hasNoFixedIssues()
+                .hasReferenceBuildId(StringUtils.EMPTY);
+    }
+
+    @Test
+    void shouldHaveCorrectReports() {
+        Run run = mock(Run.class);
```

Comment:
`Run<?, ?> run` then the IntelliJ warning disappears.

Reply:
Thanks, I didn't know that!